subprojects/packagefiles/libtiff/libtiff/meson.build

```diff
@@ -65,11 +65,12 @@ if host_machine.system() == 'windows'
 endif
 
 tiff4_lib = library(
-  'tiff4',
```

Comment:
Does this impact the generated pkgconfig file?

Reply:
The filename is the same, and the only change to the contents is the `-l` argument:

```patch
@@ -6,6 +6,6 @@
 Description: Tag Image File Format (TIFF) library.
 Version: 4.6.0
 Requires.private: libjpeg, liblzma, zlib
-Libs: -L${libdir} -ltiff4
+Libs: -L${libdir} -ltiff
 Libs.private: -lm
 Cflags: -I${includedir}
```